### PR TITLE
103a following topics

### PR DESCRIPTION
**Pull request by Lepozepo**
Fixes [this](https://github.com/NewSpring/Apollos/issues/103)

### CODE REVIEW

**Comment by Lepozepo** [comment]
Basic queries and mutations are working, need to integrate it with home feed next before we merge
newspring

**Comment by newspringweb** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Lepozepo** [comment]
@conrad-vanl I added the tests you requested

**Review by conrad-vanl — APPROVED** [review]

File: src/rock/models/feeds/resolver.js
Comment (conrad-vanl) [inline src/rock/models/feeds/resolver.js:17]: @Lepozepo Would it be easy enough to just change this logic to be something like:

```
let channels = get(opts, 'content.channels');
if (!channels) {
  const topics = await models.User.getUserFollowingTopics(person && person.PrimaryAliasId);
  channels = topics
    .map ...
    ...
}

Comment (Lepozepo) [reply src/rock/models/feeds/resolver.js:17]: For backwards compatibility? Yeah, that would work

Comment (conrad-vanl) [reply src/rock/models/feeds/resolver.js:17]: yes

**Pull request merged**
